### ISSUE TITLE
New time lapse enabled by default and produces large mp4 files

### ISSUE THREAD

**Issue by aitov** [CroatianMeteorNetwork/RMS#488]
Hi,
Recently was introduced new time lapse functionality which enabled by default even if I don't have it in my config.
As result:

- it creates very large  bz2 archives (>1 Gb for Full HD) even if no sky at night (also I'm not sure why we need upload it to server)
- consumes more space on disk and it leads to save less history in captured files
- it wrongly calculates required free space and we have corrupted large mp4 files which could not be played
- for small sd cards (<=128gb) it even does not create mp4 file but leaves images in FrameFiles 

So my proposition : disable it by default

**Comment by Cybis320** [comment]
I agree that it's a major issue. I see that the tar.bz2 with the frame files are created by default. There are options to delete the frame files programmatically after the mp4 creation.
https://github.com/CroatianMeteorNetwork/RMS/blob/e87c8080af987b2b0ed3dcca5ee616aa54f6fcc6/RMS/Reprocess.py#L544
We would need to change to cleanup='delete'. Ideally, make it a config option defaulted to delete.

Ideally, we want to keep making these frame mp4s for the climate research project. I think at a minimum we need to default to deleting the frame jpg after mp4 creation, which would save a GB of space per night.
